Resources/js/translator.js

```diff
@@ -149,6 +149,11 @@
             );
 
             var _number  = parseInt(number, 10);
+            parameters = parameters || {};
+
+            if (!parameters.count) {
```

Comment:
Please also add tests covering the case where the count is provided explicitly too